boxoffice/models/discount_policy.py

```diff
@@ -43,6 +45,12 @@ class DiscountPolicy(BaseScopedNameMixin, db.Model):
     percentage = db.Column(db.Integer, nullable=True)
     # price-based discount
     is_price_based = db.Column(db.Boolean, default=False, nullable=False)
+
+    # Note: discount_code_base was added in the initial migration,
+    # but was removed from the model for a period of time
```

Comment:
This comment should be in the migration to explain why the column is not being added there.